app/main/views/frameworks.py

```diff
@@ -216,16 +216,21 @@ def get_framework_suppliers(framework_slug):
 
     supplier_frameworks = SupplierFramework.query.filter(
         SupplierFramework.framework_id == framework.id
-    )
+    ).outerjoin(SupplierFramework.framework_agreements)
 
     if agreement_returned is not None:
         if convert_to_boolean(agreement_returned):
             supplier_frameworks = supplier_frameworks.filter(
-                SupplierFramework.agreement_returned_at.isnot(None)
-            ).order_by(SupplierFramework.agreement_returned_at.desc())
+                SupplierFramework.agreement_returned_at.isnot(None) |
```

Comment:
Is there a reason for using the bitwise `|` and `&` operators rather than `or` and `and`? I can see they will work OK here, but so would the regular boolean operators I think?